sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationAutomatedLiveTestBase.java

```diff
@@ -278,11 +281,8 @@ protected String parseIdsFromIdentifier(CommunicationIdentifier communicationIde
             : removeAllNonChar(communicationIdentifierModel.getRawId());
     }
 
-    /* Change the plus + sign to it's unicode without the special characters i.e. u002B.
-     * It's required because the dispatcher app receives the incoming call context for PSTN calls
-     * with the + as unicode in it and builds the topic id with it to send the event.*/
     protected static String removeAllNonChar(String input) {
-        return input.replace("+", "u002B").replaceAll("[^a-zA-Z0-9_-]", "");
+        return input.replaceAll("[^a-zA-Z0-9_-]", "");
```

Comment:
Why the first replace is not needed any more?

Reply:
this was a bug in dispatcher that was addressed by redployment. This special treatment is now removed as dispatcher correctly parses MRI